### PR TITLE
Update Springer Link

### PR DESCRIPTION
**Pull request by adam3smith**
only relies on RIS now with some supplementing. Tests look very good.

### CODE REVIEW

**Comment by adam3smith** [comment]
@zuphilip -- let me know if you'll be able to review this

**Comment by adam3smith** [comment]
https://forums.zotero.org/discussion/73189/zotero-chrome-plug-in-overriding-editors-in-book-sections-chapters#latest

File: Springer Link.js [hunk 1]
```diff
@@ -89,13 +89,19 @@ function complementItem(doc, item) {
 					or @id="abstract-about-doi"][1]'
 		);
 	}
+	if (!item.language) {
+		item.language = ZU.xpathText(doc, '//meta[@name="citation_language"]/@content');
+	}	
 	if(!item.publisher) {
```
Comment (zuphilip) [inline Springer Link.js]: I prefer to add a space after the `if` as you did here (but there is quite a mismatch in that file). Can we agree that we generally want that? See also https://eslint.org/docs/rules/keyword-spacing

Comment (adam3smith) [reply Springer Link.js]: I believe the space is standard Zotero style, too, so yes, let's do that.

**Comment by adam3smith** [comment]
Thanks!
Do you want to add the eslint rules to the built-in tests?

File: Springer Link.js (hunk 1 above)
Comment (zuphilip) [reply Springer Link.js]: Great!

> Unsure whether to address that retrospectively, though.

Should be possible to replace this by simple search and replace. I will try to look into this...

**Comment by zuphilip** [comment]
We are using currently JSHint. Thus, we would need to change to ESLint before we can add the rule there. I can see whether a PR is easy or maybe a first open a issue.

**Pull request merged**